lib/pages/chat/events/html_message.dart

```diff
@@ -34,15 +34,34 @@ class HtmlMessage extends StatelessWidget {
     // We do *not* do this in an AST and just with simple regex here, as riot-web tends to create
     // miss-matching tags, and this way we actually correctly identify what we want to strip and, well,
     // strip it.
-    final renderHtml = html.replaceAll(
-      RegExp(
-        '<mx-reply>.*</mx-reply>',
-        caseSensitive: false,
-        multiLine: false,
-        dotAll: true,
-      ),
-      '',
-    );
+    final renderHtml = html
+        .replaceAll(
+          RegExp(
+            '<mx-reply>.*</mx-reply>',
+            caseSensitive: false,
+            multiLine: false,
+            dotAll: true,
+          ),
+          '',
+        )
+        .replaceAll(
```

Comment:
I don't think we need two replaceAll methods here. With regex it should be possible to do it in one by selecting the content inside of tg-forward.